index.html

```diff
@@ -72,6 +78,7 @@ <h3 class="title flex">I'am a Software Developer</h3>
       <h2 class="work-header">My recent Works</h2>
       <div class="works flex">
         <div class="img-works"></div>
+        <div class="work-container">
```

Comment:
- [ ]  OPTIONAL: The width proportions of the `img-works` and `work-container` classes inside the works element is not as the original design. Please see the attached images for comparison. 
- [ ] OPTIONAL: Color differences of some element is also there. 
- [ ] OPTIONAL: Same is true for the `btn-flex` class child elements. 
![image](https://user-images.githubusercontent.com/64894927/121424266-d2cfed80-c979-11eb-80bc-afe566871593.png)   
![image](https://user-images.githubusercontent.com/64894927/121424313-e11e0980-c979-11eb-9b24-ef64b4edff58.png)